src/components/slider/index.tsx

```diff
@@ -327,7 +327,7 @@ export default class Slider extends PureComponent<SliderProps, State> {
 
       const activeOrInactiveStyle = !this.props.disabled ? (start ? activeStyle : style) : {};
       this._thumbStyles.style = _.omit(activeOrInactiveStyle, 'height', 'width');
-      this.thumb.setNativeProps(this._thumbStyles);
+      this.thumb.current?.setNativeProps(this._thumbStyles);
```

Comment:
Same here, in the check above you probably need to check `this.thumb.current` 
Technically `this.thumb` is never undefined cause it is `{current: undefined}` when initialized